src/internal/m365/collection/drive/url_cache.go

```diff
@@ -201,6 +207,7 @@ func (uc *urlCache) readCache(
 
 	props, ok := uc.idToProps[itemID]
 	if !ok {
+		uc.counter.Inc(count.URLCacheMiss)
```

Comment:
[This](https://github.com/alcionai/corso/blob/1cbabdb73a14854fc97fad87f835ab1edac5e7b7/src/internal/m365/collection/drive/collection.go#L361) would be a better place to count `URLCacheMiss` counter.

Reply:
i think both have a place, each with a dfferent stat.  I've added an `ItemDownloadUrlRefetch` key inc at your proposed location.